### PR TITLE
TS minor fixes

### PR DESCRIPTION
**Pull request by dron0gus**

### CODE REVIEW

**Comment by dron0gus** [comment]
f0_module failed due to broken master, not this PR.

**Comment by rusefillc** [comment]
just created https://github.com/mck1117/wideband/issues/179

File: firmware/console/binary/tunerstudio_io_serial.cpp [hunk 1]
```diff
@@ -179,9 +179,9 @@ int SerialTsChannel::start(uint32_t baud) {
 		chThdSleepMilliseconds(10);
 
 		/* now reset BT to apply new settings */
-		chprintf((BaseSequentialStream *)m_driver, "AT+RESET\r\n", baudRateCodes[baudIdx]);
+		chprintf((BaseSequentialStream *)m_driver, "AT+RESET\r\n");
 		if (bt_wait_ok() != 0) {
-			return -1;
+			goto error;
```
Comment (mck1117) [inline firmware/console/binary/tunerstudio_io_serial.cpp:184]: please do not use goto as it can break c++ programs

**Comment by rusefillc** [comment]
@dron0gus please rebase :)

**Comment by dron0gus** [comment]
@mck1117 rebased, fixed.

**Comment by rusefillc** [comment]
@mck1117 poke

**Pull request merged**